cql/DashboardLibrary.cql

```diff
@@ -124,6 +124,11 @@ code "Histiocytes": '14295007' from "SNOMED-CT" display 'Resident tissue macroph
 code "AGC Favor Neoplasia": '373883009' from "SNOMED-CT" display 'Atypical glandular cells, favor neoplastic (morphologic abnormality)'
 code "Endocervical Cells Favor Neoplasia": '373882004' from "SNOMED-CT" display 'Atypical endocervical cells, favor neoplastic (morphologic abnormality)'
 code "Endocervical Curettage": '52889002' from "SNOMED-CT" display 'Endocervical curettage (procedure)'
+code "Atypical Glandular Cells Uncertain": '103639009' from "SNOMED-CT" display 'Atypical glandular cells of uncertain significance, probably benign (morphologic abnormality)'
+code "Atypical Endometrial Cells": '103646000' from "SNOMED-CT" display 'Atypical endometrial cells of undetermined significance (morphologic abnormality)'
+code "Unsatisfactory": '126481000119106' from "SNOMED-CT" display 'Vaginal Papanicolaou smear unsatisfactory for evaluation (finding)'
+code "Absent Transformation Zone": '412716005' from "SNOMED-CT" display 'Cervical smear transformation zone cells absent (situation)'
+code "Endocervical AIS": '447760009' from "SNOMED-CT" display 'Endocervical adenocarcinoma in situ (disorder)'
```

Comment:
Should we avoid repeating code declarations between Dashboard Library and Management Library? I might suggest that if we want to move codes to DashboardLibrary, it is safe to remove them from ManagementLibrary.

Reply:
I meant the ManageRareAbnormality.cql file. Here you add a number of codes that were already defined in [ManageRareAbnormality.cql](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ManageRareAbnormality.cql#L41-L60). If you're defining them in DashboardLibrary.cql here, then you no longer need to redefine them in ManageRareAbnormality.cql where they are actually used, since you can just refer to the code already defined in DashboardLibrary.cql.

Reply:
Outstanding comment. When these codes that you added here in DashboardLibrary.cql are used in ManageRareAbnormality.cql, they can now reference this Dashboard version of the code, using `Dash."______"`. Therefore, these codes do not need to be declared again in ManageRareAbnormality.cql.

We should probably have a more general conversation at some point around convention for where the code declarations for different libraries should take place and whether or not we want to do any refactoring.

Reply:
Okay. Can you create a Jira ticket?

Reply:
@mrnosal I saw you marked this comment as resolved. Have you created a Jira ticket for refactoring the codes / value sets? Can you link to it here please?

Reply:
[Refactor CQL Code to consolidate codes and value set definitions](https://jira.mitre.org/browse/CCSMCDS-142)